codeserver/ubi9-python-3.12/get_code_server_rpm.sh

```
@@ -0,0 +1,84 @@
+	cd .. && rm -rf util-macros-${UTIL_MACROS_VERSION}/
+
+	export X_KB_FILE_VERSION=1.1.3
+	curl -L https://www.x.org/releases/individual/lib/libxkbfile-${X_KB_FILE_VERSION}.tar.gz | tar xz
+	cd libxkbfile-${X_KB_FILE_VERSION}/
+	./configure --prefix=/usr && make install -j ${MAX_JOBS}
+	cd .. && rm -rf libxkbfile-${X_KB_FILE_VERSION}/
+    export PKG_CONFIG_PATH=$(find / -type d -name "pkgconfig" 2>/dev/null | tr '\n' ':')
+
+	# install nfpm to build rpm
+	NFPM_VERSION=$(curl -s "https://api.github.com/repos/goreleaser/nfpm/releases/latest" | jq -r '.tag_name') \
+	    && dnf install -y https://github.com/goreleaser/nfpm/releases/download/${NFPM_VERSION}/nfpm-${NFPM_VERSION:1}-1.$(uname -m).rpm
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)